src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/NuGetPowerShellUsageCollector.cs

```diff
@@ -0,0 +1,371 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.Common;
+using NuGet.VisualStudio.Telemetry.Powershell;
+
+namespace NuGet.VisualStudio.Telemetry
+{
+    public sealed class NuGetPowerShellUsageCollector : IDisposable
+    {
+        // PMC, PMUI powershell telemetry consts
+        public const string NuGetPMCExecuteCommandCount = nameof(NuGetPMCExecuteCommandCount);
```

Comment:
As mentioned in another comment, initializations should only have the first letter capitalized, the rest lower case, like `Sql`, `Xml, `Json`.  I see many, many instances of all upper `PMC` and `PMUI` in this file. Please check the whole PR.

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

> The PascalCasing convention, used for all identifiers except parameter names, capitalizes the first character of each word (including acronyms over two letters in length), as shown in the following examples:

> PropertyDescriptor HtmlTag

> A special case is made for two-letter acronyms in which both letters are capitalized, as shown in the following identifier:

> IOStream


This is why we have APIs like:
* [`SqlClient`](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient?view=dotnet-plat-ext-5.0), not `SQLClient`
* [`XmlDocument`](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmldocument?view=net-5.0), not `XMLDocument`
* [`JsonSerializer`](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer?view=net-5.0), not `JSONSerializer`.

So, `NuGetPMCExecuteCommandCount` should be `NuGetPmcExecuteCommandCount`. There are many examples of this naming convention not being followed in this PR for both `Pmc` and `Pmui`. When writing human text, we continue to capitalize the full "word", so I'd still talk about PMC, but code using it as part of a variable name should use `Pmc`.